src/Provider/GoogleMapsPlaces.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/**
+ * This file is part of the Geocoder package.
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @license    MIT License
+ */
+
+namespace Geocoder\Provider;
+
+use Geocoder\Exception\UnsupportedOperation;
+use Ivory\HttpAdapter\HttpAdapterInterface;
+
+/**
+ * @author Germán Mauricio Muñoz <mauricio@mapalesoftware.com>
+ */
+final class GoogleMapsPlaces extends GoogleMaps
+{
+    /**
+     * @var string
+     */
+    const ENDPOINT_URL = 'https://maps.googleapis.com/maps/api/place/textsearch/json?query=%s';
+
+    /**
+     * @param HttpAdapterInterface $adapter An HTTP adapter.
+     * @param string               $apiKey  Google Places API key
+     * @param string               $locale  A locale (optional).
+     */
+    public function __construct(HttpAdapterInterface $adapter, $apiKey, $locale = null)
```

Comment:
You should not be using Ivory\HttpAdapter\HttpAdapterInterface anymore. We use HTTPlug. See other adapters for reference.